### PR TITLE
Add Rating css

### PR DESCRIPTION
**Pull request by rahulcs754**
Added rating css for (ecommerce site)

image

### CODE REVIEW

File: css/components/navigation.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
+/*
+|| navigation css
+*/
+.header {
+    display: flex;
+    flex-direction: row;
+	justify-content: space-between;
```
Comment (tanishq20) [inline css/components/navigation.css]: use prettier to format it.

File: css/components/rating.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
+/*
+|| Rating Css 
+*/
+.rating{
+    display: flex;
+    gap: 1px;
+    padding: 1rem;
+
```
Comment (tanishq20) [inline css/components/rating.css]: Avoid giving extra spaces.

File: css/components/rating.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+.rating-badge{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    gap: 5px;
+    padding: 0.3rem 0.5rem ;
+    background-color: var(--warning-color);
+    color: var(--light-black);
+    border-radius: 30%;
```
Comment (tanishq20) [inline css/components/rating.css]: `border-radius:30%` doesn't look gook. Try with `px` or give height and width for the perfect shape.

**Review by tanishq20 — COMMENTED** [review]
Code look clean. Just a few suggestions mentioned.

- Please don't merge two different branch in the same PR.

File: css/components/rating.css [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    color: var(--warning-color);
+}
+
+.rating-item-checked{
+    color: var(--warning-color);
+}
+
+
+
+.rating-badge{
```
Comment (shuntzhere) [inline css/components/rating.css]: Remove the extra spaces

**Review by shuntzhere — APPROVED** [review]
Some extra spacings were found. Otherwise looks neat.

**Review by tanishq20 — APPROVED** [review]

**Pull request merged**